tools/build_framework.sh

```diff
@@ -79,7 +79,7 @@ case "$PUBLISH_STEP" in
         echo "---"
         echo "Build complete, skipping publish step."
         echo "Use one of the following additional arguments to get something that runs on a cluster:"
-        echo "- 'local': Host the build in a local HTTP server for use by a vagrant cluster."
+        echo "- 'local': Host the build in a local HTTP server for use by a DC/OS Vagrant cluster."
```

Comment:
strictly speaking this was written for DC/OS Docker clusters but within a vagrant container. We found DC/OS Vagrant to not really be useful for our purposes (required huge machines when we're targeting e.g. laptops with 4-8GB)

see also: https://github.com/mesosphere/dcos-commons#quick-start